### PR TITLE
docs: Exotel integration

### PR DESCRIPTION
**Pull request by surajshetty3416**
https://github.com/frappe/erpnext/pull/17763

### CODE REVIEW

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<!-- add-breadcrumbs -->
+
+# Exotel Integration
+
+This integration enables you to integrate Exotel with your ERPNext system, if you are using Exotel for business to capture the leads with their phone number directly in your ERPNext account.
```
Comment (netchampfaris) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
This integration allows you to integrate Exotel with your ERPNext system, if you are using Exotel for business to capture the leads with their phone number directly in your ERPNext account.
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+## Features
+
+1. Track incoming calls in your ERPNext account.
+2. Shows lead/customer information pop-up to employees when an incoming call is received.
+
+## How to setup
+
+### 1. Setup in your Exotel account for call popup
```
Comment (netchampfaris) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
### 1. Setup your Exotel account
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+- Login to your Exotel account and go to App Bazar
+- Create a new App for a new flow.
+- Setup the flow as you wish it to be.
+- In your connect API under "Create popup..." and paste following URL:
+`https://<your-site>/api/method/erpnext.erpnext_integrations.exotel_integration.handle_incoming_call`
+
+<img class="screenshot" alt="Connect Applet" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/connect_applet.png">
```
Comment (netchampfaris) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
![Connect Applet](/docs/assets/img/erpnext_integrations/exotel_integration/connect_applet.png)
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md (hunk 1 above)
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
This integration allows you to integrate Exotel into your ERPNext account. Leads and their phone numbers captured via Exotel will be directly saved to your ERPNext.
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<!-- add-breadcrumbs -->
+
+# Exotel Integration
+
+This integration enables you to integrate Exotel with your ERPNext system, if you are using Exotel for business to capture the leads with their phone number directly in your ERPNext account.
+
+## Features
+
+1. Track incoming calls in your ERPNext account.
+2. Shows lead/customer information pop-up to employees when an incoming call is received.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: use regular bullets not numbered

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+<!-- add-breadcrumbs -->
+
+# Exotel Integration
+
+This integration enables you to integrate Exotel with your ERPNext system, if you are using Exotel for business to capture the leads with their phone number directly in your ERPNext account.
+
+## Features
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
This integration has the following features:
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Track incoming calls in your ERPNext account.
+2. Shows lead/customer information pop-up to employees when an incoming call is received.
+
+## How to setup
+
+### 1. Setup in your Exotel account for call popup
+
+- Login to your Exotel account and go to App Bazar
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
- Login to your Exotel account and go to App Bazar.
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md (hunk 3 above)
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: he's just showing off that he's a web dev. "HTML" /jk
but seriously md image tags are simpler

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+- In your connect API under "Create popup..." and paste following URL:
+`https://<your-site>/api/method/erpnext.erpnext_integrations.exotel_integration.handle_incoming_call`
+
+<img class="screenshot" alt="Connect Applet" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/connect_applet.png">
+
+<img class="screenshot" alt="Call Popup Section" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/create_popup_section.png">
+
+> **Note:** Replace `<your-site>` in URL with your site name. For example, if the site name is **frappe.erpnext.com** then the URL will be: `https://frappe.erpnext.com/api/method/erpnext.erpnext_integrations.exotel_integration.handle_incoming_call`
+
+- After that add a Passthru applet under "After Call Conversation ends" and paste following URL:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: take the URL to a new line

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+`https://<your-site>/api/method/erpnext.erpnext_integrations.exotel_integration.handle_end_call`
+
+<img class="screenshot" alt="After Conversation Ends Section" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/after_conversation_ends_section.png">
+
+<img class="screenshot" alt="After call ends section" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/passthru_end_call.png">
+
+> **Note:** Make sure to check "Make Passthru Async"
+
+- Similary, add a Passthru applet under "If nobody answers..." section and paste following URL:
+`https://<your-site>/api/method/erpnext.erpnext_integrations.exotel_integration.handle_missed_call`
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: URL in new line

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+<img class="screenshot" alt="No Response Section" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/no_response.png">
+
+<img class="screenshot" alt="After call ends section" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/passthru_missed_call.png">
+
+> **Note:** Make sure to check "Make Passthru Async"
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
> **Note:** Make sure to check "Make Passthru Async".
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+- Save the flow.
+- Now assign this newly created app to your ExoPhone from which you receive your business calls.
+
+### 2. Setup in ERPNext
+
+- From Awesome Bar, go to 'Exotel Settings'.
+- Set your "Exotel SID" and "Exotel Token". You can find your Exotel API key and token on your [Exotel Dashboard](https://my.exotel.com/apisettings/site#api-credentials)
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]:
```suggestion
- Set your "Exotel SID" and "Exotel Token". You can find your Exotel API key and token on your [Exotel Dashboard](https://my.exotel.com/apisettings/site#api-credentials).
```

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+<img class="screenshot" alt="After call ends section" src="{{docs_base_url}}/assets/img/erpnext_integrations/exotel_integration/passthru_missed_call.png">
+
+> **Note:** Make sure to check "Make Passthru Async"
+
+- Save the flow.
+- Now assign this newly created app to your ExoPhone from which you receive your business calls.
+
+### 2. Setup in ERPNext
+
+- From Awesome Bar, go to 'Exotel Settings'.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: add regular navigation if available

Comment (surajshetty3416) [reply erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: Regular navigation is not available

File: erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md (hunk 5 above)
Comment (surajshetty3416) [reply erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: isn't it the standard structure?

Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/erpnext_integration/exotel_integration.md]: ok so add numbering to all headings

**Pull request merged**